go/pkg/cas/upload.go

```diff
@@ -113,6 +136,11 @@ type uploader struct {
 	// muLargeFile ensures only one large file is read at a time.
 	// TODO(nodir): ensure this doesn't hurt performance on SSDs.
 	muLargeFile sync.Mutex
+
+	// checkBundler bundles digests that need to be checked for presence on the
+	// server.
+	checkBundler *bundler.Bundler
+	seenHashes   sync.Map
```

Comment:
I like the idea, so I've added a TODO because it is more than just making it a global variable:
- we use isolate client on long-running servers, so having a global cache without at least LRU sounds like a memory leak.
- once we make it more global, instance name and RBE service name must be incorporated into the cache key

Reply:
not sure what happened. Added TODO.